components/ChallengeMaterial.tsx

```diff
@@ -17,10 +18,29 @@ type ChallengeMaterialProps = {
 }
 
 export const ChallengeTitleCard: React.FC<ChallengeTitleCardProps> = props => {
+  const { currentState: state, active } = props
+  let cardStyles = []
+  if (state === 'complete') cardStyles.push('challenge-title-card--done')
```

Comment:
could you use `{}` please? Its easier to read / less error prone in the future. Especially since there are quite a few piece of logic.

```
if (...) {
  ...
} else {
  ...
}
if (...) {
  ...
}
```

Reply:
Fixed!